playbooks/install.yml

```diff
@@ -0,0 +1,10 @@
+# Playbook for installing the EESSI compatibility layer.
+---
+- name: Install compatibility layer
+  hosts: cvmfsstratum0servers
+  environment:
+    PATH: "{{ prefix_location }}/usr/bin:{{ ansible_env.PATH }}"
```

Comment:
Definitely makes sense to add /bin, I think I even ran into an issue with bzip2 recently because /bin was not in my path.
I'm not sure if EPREFIX and BPREFIX are really necessary (haven't run into issues), but I think it's indeed better to have them defined anyway. So I'll add/update these variables.